README.md

```diff
@@ -1,12 +1,46 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
-<!-- Copyright Contributors to the Egeria project. -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
```

Comment:
I notice our newer files drop the ODPi (since we moved to LF AI & Data) - this new form was in the file before change? that being said almost all still use old format, so I think this is just fine, and can skip any change here